### PR TITLE
Draft working guide for LiteFS integration with multiregion Laravel S…

### PR DESCRIPTION
**Pull request by KTanAug21**
…QLite Fly App

### CODE REVIEW

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+---
+
+So you've decided to give SQLite a chance huh? If your answer's a bold, brave <b><i>yes</i></b>, then welcome! There's priceless <b>walkthough treasures</b> waiting for your read below! 
+
+On the other hand, if your answer is a resounding, mundane no, you might want to rethink your answer champ. You <i>are</i> here for a reason, so read on!
+
+
+## _Global SQLite--But How?_
+ 
+Admitably, SQLite's <i>a bit</i> embedded, and you might think there's nary a quick way in getting your Laravel-SQLite Fly App running multiple instances, oceans apart, running in different regions with all their wonderful data in-sync.
```
Comment (Johannes-Fly) [inline laravel/advanced-guides/global-sqlite-litefs.html.md.erb:14]: Admittedly instead of Admitably ;)

Comment (KTanAug21) [reply laravel/advanced-guides/global-sqlite-litefs.html.md.erb:14]: yooo @Johannes-Fly finally! cool, will fix this one, thanks!

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,307 @@
+---
+title: Global SQLite with LiteFS
+layout: framework_docs
+objective: Go LiteFS-global with your SQLite-backed Laravel Fly App!
+---
+
+So you've decided to give SQLite a chance huh? If your answer's a bold, brave <b><i>yes</i></b>, then welcome! There's priceless <b>walkthough treasures</b> waiting for your read below!
```
Comment (fideloper) [inline laravel/advanced-guides/global-sqlite-litefs.html.md.erb:7]:
```diff
- waiting for your read below
+ waiting for you to read below
```

(Or maybe something like `waiting for your viewing pleasure below`)

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,307 @@
+---
+title: Global SQLite with LiteFS
+layout: framework_docs
+objective: Go LiteFS-global with your SQLite-backed Laravel Fly App!
+---
+
+So you've decided to give SQLite a chance huh? If your answer's a bold, brave <b><i>yes</i></b>, then welcome! There's priceless <b>walkthough treasures</b> waiting for your read below! 
+
+On the other hand, if your answer is a resounding, mundane no, you might want to rethink your answer champ. You <i>are</i> here for a reason, so read on!
```
Comment (fideloper) [inline laravel/advanced-guides/global-sqlite-litefs.html.md.erb:9]: I love "champ" 😂

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+<small>
+What's up above? It's simply the `fly launch` command with a bunch of prompts pre-configured:
+<ul>
+<li>Specify the region as `ams` with the `--region` argument</li>
+<li>Chill and don't deploy the app yet, include the `--no-deploy` flag</li>
+</ul>
+</small>
+
+<aside class="callout">
+Savy! You should now have a pretty good `fly.toml` configured for your Laravel Application. Let's proceed below with setting your application to be deployed in more than one region.
```
Comment (fideloper) [inline laravel/advanced-guides/global-sqlite-litefs.html.md.erb:53]: I think it's `Savvy` (two `v`'s)

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+
+<%= partial "docs/laravel/partials/volume_storage_script" %>
+
+<b>On to the content of the Start Up script:</b>
+```bash
+FOLDER=/var/www/html/storage/app
+if [ ! -d "$FOLDER" ]; then
+    echo "$FOLDER is not a directory, copying storage_ content to storage"
+    cp -r /var/www/html/storage_/. /var/www/html/storage
+    echo "deleteing storage_..."
```
Comment (fideloper) [inline laravel/advanced-guides/global-sqlite-litefs.html.md.erb:132]: `deleteing` -> `deleting`

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb (hunk 2 above)
Comment (KTanAug21) [reply laravel/advanced-guides/global-sqlite-litefs.html.md.erb:7]: right-o! I need to up my skills on clear writing, thanks @fideloper!

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb (hunk 3 above)
Comment (KTanAug21) [reply laravel/advanced-guides/global-sqlite-litefs.html.md.erb:9]: I tossed my coin and experimented adding some tone to my narration in this doc, I'm super glad you noticed!!

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb (hunk 5 above)
Comment (KTanAug21) [reply laravel/advanced-guides/global-sqlite-litefs.html.md.erb]: fixed! thanks!

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb (hunk 4 above)
Comment (KTanAug21) [reply laravel/advanced-guides/global-sqlite-litefs.html.md.erb]: fixed!

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb (hunk 2 above)
Comment (KTanAug21) [reply laravel/advanced-guides/global-sqlite-litefs.html.md.erb]: fixed!

File: laravel/advanced-guides/global-sqlite-litefs.html.md.erb (hunk 1 above)
Comment (KTanAug21) [reply laravel/advanced-guides/global-sqlite-litefs.html.md.erb]: fixed!

**Review by fideloper — APPROVED** [review]

**Pull request merged**